### DISCUSSION TITLE
Swagger and Lucid Mongodb

### DISCUSSION THREAD

**Discussion by mohamedhashem-98** [adonisjs/core#1422] [Help]
Hello , 
 - Is v5 supported with swagger like v4 ?
 - Is lucid mongo supported in v5 ?
 how can integrate them and Thanks .

**Comment by McSneaky** [comment]
Sounds like should rise issue in related packages repos

**Reply by gabrieldocs** [reply]
@McSneaky could you point any issue that relates to that?

**Reply by McSneaky** [reply]
https://github.com/duyluonglc/lucid-mongo/issues/267

It's a community package
